packs/equipment-effects/effect-drake-rifle-cold.json

```
@@ -0,0 +1,45 @@
+{
+    "_id": "s79LDAwEZex6sy14",
+    "img": "icons/magic/water/barrier-ice-water-cube.webp",
+    "name": "Effect: Drake Rifle (Cold)",
+    "system": {
+        "description": {
+            "value": "<p><strong>Effect applied by</strong>@UUID[Compendium.pf2e.equipment-srd.Item.Drake Rifle (Cold)]</p><hr /><p><span>Frozen spittle clings to the target. The target takes a –10-foot status penalty to their Speeds for 1 minute.</span></p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Guns & Gears"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Drake Rifle (Cold)",
+                "selector": "speed",
+                "slug": "effect-drake-rifle-cold",
+                "type": "status",
+                "value": -10
+            }
```

```suggestion
            {
                "key": "FlatModifier",
                "selector": "all-speed",
                "type": "status",
                "value": -10
            }
```
We have to localize any labels or text in rule elements. We also want to be as specific as possible with the selector for speeds to make sure there's no ambiguity between what speed is being affected. Also we don't need to provide a slug.